### PR TITLE
Feature/postgresql 9.5

### PR DESCRIPTION
**Pull request by CloudNiner**
Notes are in the first commit's log.

### CODE REVIEW

**Comment by CloudNiner** [comment]
I'm currently testing my instructions in the commit log for upgrading an existing vm. Provisioning is slow. Will tweak the instructions provided as necessary if things turn sour.

EDIT: Reprovision following update instructions in commit log was successful.

File: ansible/roles/cartodb.postgresql/tasks/postgis.yml [hunk 1]
```diff
@@ -4,8 +4,10 @@
    apt: name={{ item }} state=present
    with_items:
     - libxml2-dev
-    - postgresql-9.3-postgis-2.2
-    - postgresql-9.3-postgis-scripts
+    - liblwgeom-2.1.8
```
Comment (hectcastro) [inline ansible/roles/cartodb.postgresql/tasks/postgis.yml]: This was causing a fresh install to bomb. Do you know why this and `postgis` are needed in addition to the last two packages?

Comment (CloudNiner) [inline ansible/roles/cartodb.postgresql/tasks/postgis.yml]: Interesting. No I do not know. These packages are listed as dependencies on the cartodb docs. I'll reproduce and investigate a bit.

**Comment by CloudNiner** [comment]
Updated to use master branch for each cartodb repo, since this is intended as a development box. Was able to provision successfully with the group_vars/all.example file currently on this branch. Let me know if you run into any other issues.

Once merged, I'm going to cut a new release with an updated changelog describing these changes along with backwards compatibility with older versions of cartodb.

**Comment by hectcastro** [comment]
👍

**Pull request merged**